scripts/multi-arch.yaml

```diff
@@ -0,0 +1,10 @@
+image: amazon/amazon-ecs-agent:${tag}
```

Comment:
It's rather misleading to use `${tag}` here as though an environment variable named `$tag` would be respected here. I realize that the string will later be replaced by sed (this string makes that command rather unwieldy) so please pick something else.. maybe `TAG` and replace that? There are other tools that can replace environment variables in text files, but let's stick with `sed`.